strawberry/enum.py

```diff
@@ -22,6 +22,8 @@ def _process_enum(cls, name=None, description=None):
         values=[(item.name, GraphQLEnumValue(item.value)) for item in cls],
         description=description,
     )
+    # :D
+    cls.field._strawberry_type = cls
```

Comment:
this is needed to fetch the strawberry type from the graphql field, we can find a way to improve it (maybe our own type map)